src/profile.jl

```diff
@@ -23,7 +23,7 @@ function Base.copy(ret::ReMat{T,S}) where {T,S}
         copy(ret.λ),
         ret.inds,
         ret.adjA,
-        copy(ret.scratch))
+        copy(ret.scratch)) # likely don't need to copy scratch, as it is temporary storage.
```

Comment:
I copied it to guarantee that we would never have a race condition: we're not currently doing any relevant threading on these things, but I could see that happening one day

Reply:
Seems reasonable.  Thanks for responding.